packages/web/components/PostEditor/PostEditor.tsx

```diff
@@ -224,6 +228,16 @@ const PostEditor: React.FC<PostEditorProps> = ({
           topics={postTopics}
         >
           <div className="header-preview-options">
+            <Button
+              type="button"
+              variant={ButtonVariant.Secondary}
+              className="unsplash-button"
+              onClick={(): void => {
```

Comment:
You should be able to remove these `(): void => ...` type annotations in these inline functions. Generally we let TypeScript figure things out where it can reliably interpret things